assets/src/edit-story/panels/fullbleed.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ActionButton, Panel, Title, getCommonValue } from './shared';
+
+function FullbleedPanel( { selectedElements, onSetProperties } ) {
+	// The x/y/w/h/r are kept unchanged so that toggling fullbleed will return
+	// the element to the previous non-fullbleed position/size.
+	const isFullbleed = getCommonValue( selectedElements, 'isFullbleed' );
+	const [ state, setState ] = useState( { isFullbleed } );
+	useEffect( () => {
+		setState( { isFullbleed } );
+	}, [ isFullbleed ] );
+	const handleClick = ( evt ) => {
+		const newState = { isFullbleed: ! state.isFullbleed };
+		setState( newState );
+		onSetProperties( newState );
+		evt.preventDefault();
```

Comment:
why is preventDefault needed here?

Reply:
Removed. Not needed. Just copy/pasted it from another panel that used `submit` event making the `preventDefault` necessary. But no need in this case.